lib/inferno_template/metadata.rb

```diff
@@ -5,14 +5,89 @@ class Metadata < Inferno::TestKit
     id :inferno_template
     title 'Inferno Template'
     description <<~DESCRIPTION
-      This is a big markdown description of the test kit.
+      **TODO**: 1-2 sentence description of the purpose of the test kit
+
+      E.g.:
+
+      > The ONC Certification (g)(10) Standardized API Test Kit is a testing tool for Health IT systems seeking to meet the requirements of the Standardized API for Patient and Population Services criterion § 170.315(g)(10) in the ONC Certification Program. It is an approved test method for the § 170.315(g)(10) test procedure.
```

Comment:
I guess I would be a little more comfortable if we put in something like US Core in here vs. a certification test kit, just because publishing policy-looking documentation makes it look like official content vs. just an example.  Or maybe just replacing it with a 'dummy' certification program / criteria.  E.g. The Inferno Template Test Kit is a testing tool for Health IT systems seeking to meet the requirements of the API Criterion within the Example Certification Program.